doc/customizing/client-side-config.md

```diff
@@ -45,7 +45,7 @@ Specifies various options for configuring TerriaJS:
 |`appName`|no|**string**|`"TerriaJS App"`|TerriaJS uses this name whenever it needs to display the name of the application.|
 |`supportEmail`|no|**string**|`"info@terria.io"`|The email address shown when things go wrong.|
 |`defaultMaximumShownFeatureInfos`|no|**number**|`100`|The maximum number of "feature info" boxes that can be displayed when clicking a point.|
-|`regionMappingDefinitionsUrl`|yes|**string**|`"build/TerriaJS/data/regionMapping.json"`|URL of the JSON file that defines region mapping for CSV files. This option only needs to be changed in unusual deployments. It has to be changed if deploying as static site, for instance.|
+|`regionMappingDefinitionsUrl`|yes|**string | string[]**|`"build/TerriaJS/data/regionMapping.json"`|URL of the JSON file that defines region mapping for CSV files. This option only needs to be changed in unusual deployments. It has to be changed if deploying as static site, for instance. It supports multiple URLs - if provided then the first matching region will be used (in order of URLs) |
```

Comment:
Deprecate regionMappingDefinitionsUrl and create a new parameter for multiple URLs

Reply:
Done!